From 8e60a4af3788745dc677d9a8f90ad352005169be Mon Sep 17 00:00:00 2001
From: Dabo Ross <daboross@daboross.net>
Date: Sat, 26 Apr 2014 14:52:22 -0700
Subject: [PATCH] Make input.server equal to conn.name rather than conn.server.
 Since conn.name is the connections unique identifier, not conn.server, I
 thought this would fit.

This does change the log directory from using the server IP to server name, but I think that's a good thing.

diff --git a/core/main.py b/core/main.py
index 0878ec9..cbabbcb 100644
--- a/core/main.py
+++ b/core/main.py
@@ -26,6 +26,7 @@ class Input:
     :type text: str
     :type match: re.__Match
     :type server: str
+    :type server_nice_name: str
     :type lastparam: str
     :type chan: str
     """
@@ -77,9 +78,11 @@ class Input:
             self.inp = self.paramlist
 
         if self.conn is not None:
-            self.server = conn.server
+            self.server = conn.name
+            self.server_nice_name = conn.nice_name
         else:
             self.server = None
+            self.server_nice_name = None
 
         if self.paramlist:
             self.chan = paramlist[0].lower()
-- 
1.9.2

